src/CodeCoverage.php

```diff
@@ -960,18 +961,18 @@ private function initializeData(): void
             $data     = [];
             $coverage = $this->driver->stop();
 
-            foreach ($coverage as $file => $fileCoverage) {
+            foreach ($coverage as $file => $lines) {
                 if ($this->filter->isFiltered($file)) {
                     continue;
                 }
 
-                foreach (\array_keys($fileCoverage) as $key) {
```

Comment:
I'm not sure why you are using array_keys() here. I suppose because you don't want to modify the array while iterating on it? some static analysis check?